kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -28,12 +28,35 @@ function unassignedContentGuard() {
   return;
 }
 
+function allowGuestAccessGuard() {
```

Comment:
We need to do this for all the class routes too, right?

Might be best just to do this as a global guard on the router itself?

We do this for `afterEach` here: https://github.com/learningequality/kolibri/blob/release-v0.14.x/kolibri/plugins/learn/assets/src/app.js#L23

So could add `beforeEach` for this check in the same place.

Noting that I didn't do that for the content unavailable, as the behaviour is differential between class and non-class routes.